README.md

```diff
@@ -37,7 +37,58 @@ Live-Ask is a product developed and maintained by [Rustunit.com](https://rustuni
 See detailed [changelog](CHANGELOG.md).
 
 ## Build & Usages Details
-(coming soon)
+**Requires Three Terminal Tabs/Instances**
+### Initial Setup
+#### Pre-requisites 
+```
+rustup update
```

Comment:
maybe also a link to a `how to install rust` source

Reply:
Will do now